libbeat/publisher/pipeline/stress/stress_test.go

```diff
@@ -68,7 +81,7 @@ func configTest(t *testing.T, typ string, configs []string, fn func(t *testing.T
 	for _, config := range configs {
 		config := config
 		t.Run(testName(typ, config), func(t *testing.T) {
-			t.Parallel()
+			// t.Parallel()
```

Comment:
TODO: restore capability to run tests in parallel (will need randomized file names)